src/aind_morphology_utils/converters.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from pathlib import Path
+
+import numpy as np
+import h5py
+import nrrd
+
+
+class NRRDToHDF5:
```

Comment:
Not sure why we need this to save it again as HDF5? Can we not load from the nrrd directly in coordinate_mapping.py ?

Reply:
This is essentially a port of the matlab scripts we used for the MouseLight registration pipeline. We converted the nrrd displacement field to HDF5, which was then uploaded to an internal database used for transforming swc files. I am not 100% sure why the resave was done, but implemented it to be consistent with the old pipeline. I agree that resaving as HDF5 may not be necessary, though it could be useful to save the displacement field in a cloud friendly chunked format such as OME-Zarr, since the displacement field can be 10s of GBs if we use the 10um atlas. It would be possible to apply transformations in the cloud without downloading a copy of the nrrd or HDF5 file. What do you think?

Reply:
Ok - yeah i had a feeling it had somethign to do with the old pipeline. OME zarr definitely seems like a good idea. It can be very useful to look at the fields.